app/templates/components/forms/user-payment-info-form.hbs

```diff
@@ -0,0 +1,49 @@
+<form class="ui form" {{action 'submit' on='submit'}}> 
+  <h3 class="ui header">
+    {{t 'Payment Information'}}
+  </h3>
+  <div class="field">
+    <label class="required" for="contactName">{{t 'Contact Name'}}</label>
+    {{input type='text' id='contactName' value=(mut userBillingInfo.billingContactName)}}
+  </div>
+  <div class="field">
+    <label class="required" for="phone">{{t 'Phone'}}</label>
+    {{input type='text' id='phone' value=(mut userBillingInfo.billingPhone)}}
```

Comment:
I think the value attribute for `{{input}}` helper should only be like `value=userBillingInfo.billingPhone` ?

Reply:
@mrsaicharan1 Done!